openapi.yaml

```
@@ -3306,15 +3326,15 @@ paths:
             application/json:
               schema:
                 properties:
-                  '': { type: string, example: 'Backup execution deleted.' }
+                  message: { type: string, example: 'Backup execution deleted.' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)